src/styles/partials/components/_summary-list.scss

```diff
@@ -0,0 +1,67 @@
+.dg_summary-list {
+  color: $gray;
+  background-size: cover;
+  background-color: $gray-lightest;
+}
+  .dg_summary-list-items {
```

Comment:
this would just change with the suggestion above.

Reply:
done